testdata/google/events/cloud/pubsub/v1/MessagePublishedData-simple.json

```
@@ -0,0 +1,5 @@
+{
+    "data": "SGVsbG8gV29ybGQh",
+    "messageId": "3103425958877813",
+    "publishTime": "2021-10-01T20:37:27.386Z"
+}
```

```suggestion
{
  "message": {
    "data": "SGVsbG8gV29ybGQh",
    "messageId": "3103425958877813",
    "publishTime": "2021-10-01T20:37:27.386Z"
  }
}
```

There are two problems this suggestion corrects:

1. (major) The data must be wrapped in a "message" object to conform with the schema.
2. (minor) We standardize on 2 space indentation.

---

Thank you, Adam. This change had been initially suggested to refer to in the documentation. Later on, it was decided to add the appropriate examples to a different folder for documentation: https://github.com/googleapis/google-cloudevents/tree/main/examples. So I will be closing this PR.